src/services/riotApiClient.js

```diff
@@ -92,15 +95,28 @@ class RiotApiClient {
       return this.transformMatchInfo(response.data)
     } catch (err) {
       logger.error(
-        `An error occured attempting to fetch the Match Data from ${response.config.url}, ${err.message}`
+        `An error occured attempting to fetch the Match Data, ${err.message}`
       )
       throw err
     }
   }
 
+  async initialize(){
+    try 
+    {
+      const latestPatchVersion = await this.fetchLatestLolPatchVersion()
+      return this.latestPatchVersion = latestPatchVersion
```

Comment:
You don't need to return this, just set it. Nobody expects a return value from calling an initialize() function.